### PR TITLE
Update longest_common_subsequence.java

### PR DESCRIPTION
**Pull request by md-shamim-ahmad**
**Fixes issue:**

**Changes:**

Reduced code size and maintained clean code format

### CODE REVIEW

File: code/dynamic_programming/src/longest_common_subsequence/longest_common_subsequence.java [hunk 1]
```diff
@@ -1,35 +1,29 @@
...
+        LongestCommonSubsequence lcs = new LongestCommonSubsequence();
+        String s1 = "AGGAGTCTAGCTAB";
+        String s2 = "AGXGTTXAYBATCGAT";
+
+        char []X = s1.toCharArray();
+        char []Y = s2.toCharArray();
+        int m = X.length;
+        int n = Y.length;
+
+        System.out.println("Length of LCS is" + " " + lcs.longestCommonSubsequence(X, Y, m, n));
```
Comment (AdiChat) [inline code/dynamic_programming/src/longest_common_subsequence/longest_common_subsequence.java:27]: Function definition is different. Make sure your code compiles and works correctly.

Comment (md-shamim-ahmad) [reply code/dynamic_programming/src/longest_common_subsequence/longest_common_subsequence.java]: @AdiChat  thanks for your review.
I fix the error.
Now please check the code.

**Comment by md-shamim-ahmad** [comment]
@AdiChat thanks for your review.
I fixed the error.
Now please check the code.

**Comment by md-shamim-ahmad** [comment]
@AdiChat Please review this PR.

**Pull request closed without merging**